specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples/SecurityPolicies_Patch.json

```diff
@@ -6,26 +6,28 @@
     "profileName": "profile1",
     "securityPolicyName": "securityPolicy1",
     "securityPolicyProperties": {
```

Comment:
This name has been changed to `securityPolicUpdateProperties` in this PR, please change it accordingly